hipcheck/src/policy_exprs/token.rs

```diff
@@ -34,6 +41,12 @@ pub enum Token {
 	#[regex(r"([1-9]?[0-9]*)", lex_integer, priority = 20)]
 	Integer(i64),
 
+	#[regex(r"[0-9]{3,4}[^\s\)]+", lex_datetime)]
+	DateTime(Zoned),
+
+	#[regex(r"P[a-zA-Z0-9]+", lex_span)]
+	Span(Span),
+
 	#[regex("([a-zA-Z]+)", lex_ident)]
 	Ident(String),
```

Comment:
Can we update the identifier regex to expect single quotes? This way there isn't a collision between `P` starting a span and being the first letter of something intended as a string identifier

Reply:
Oh right, I meant to do that. :)